### PR TITLE
Update Changelog.md

### PR DESCRIPTION
**Pull request by ClosedSourcerer**
Release Datum muss noch angepasst werden

### CODE REVIEW

**Review by lanfrans — APPROVED** [review]

File: docs-kits/kits/Demand and Capacity Management Kit/changelog.md [hunk 1]
```diff
@@ -12,7 +12,23 @@ toc_max_heading_level: 2
...
 
-## [1.1.0] - unreleased
+## [1.2.0] - unreleased
+
+### Added
+
+- Added E2E Tests to [Development View](./development-view/overview.md)
+- Added Glossary to [Adoption View](./adoption-view/overview.md)
+- Added Delta Production to [Adoption View](./adoption-view/overview.md)
+- Added Demand Fluctuation to [Adoption View](./adoption-view/overview.md)
```
Comment (Frank-Globisch) [inline docs-kits/kits/Demand and Capacity Management Kit/changelog.md:22]: depending on final agreement on terminology, "Demand Fluctuation" may be changed to "Demand Volatility" to be in sync with the DCM standard. Topic is expected to be covered this week and be discussed in our weekly.

File: docs-kits/kits/Demand and Capacity Management Kit/changelog.md [hunk 2]
```diff
@@ -12,7 +12,23 @@ toc_max_heading_level: 2
...
+- Added Load Factors to [Adoption View](./adoption-view/overview.md)
+- Added Nesting to [Adoption View](./adoption-view/overview.md)
+  
+### Changed
+
+- Updated [Core Business Process](./adoption-view/use-case/business-process.md)
```
Comment (Frank-Globisch) [inline docs-kits/kits/Demand and Capacity Management Kit/changelog.md:28]: suggested to include a bit more detailed, e.g. "Updated ... with several business examples of Demand-Capacity situations and their interpretation".

File: docs-kits/kits/Demand and Capacity Management Kit/changelog.md [hunk 3]
```diff
@@ -12,7 +12,23 @@ toc_max_heading_level: 2
...
+- Updated [Onboarding](./adoption-view/onboarding.md)
```
Comment (Frank-Globisch) [inline docs-kits/kits/Demand and Capacity Management Kit/changelog.md:29]: suggested to include a bit more detail once finally clarified; depending on outcome e.g. "Updated ... with simplified overview on actions to be taken"

**Review by Frank-Globisch — COMMENTED** [review]
see individual in-line comments.

File: docs-kits/kits/Demand and Capacity Management Kit/changelog.md (hunk 1 above)
Comment (MWADCM) [reply docs-kits/kits/Demand and Capacity Management Kit/changelog.md:22]: Included into Weekly of DCM EG at 11.10.2024
Suggestion: "Demand Fluctuation" will be replaced with "Demand Volatilities" in order to NOT touch standard.

File: docs-kits/kits/Demand and Capacity Management Kit/changelog.md (hunk 2 above)
Comment (MWADCM) [reply docs-kits/kits/Demand and Capacity Management Kit/changelog.md:28]: @Frank-Globisch Do not understand, sorry

File: docs-kits/kits/Demand and Capacity Management Kit/changelog.md (hunk 3 above)
Comment (MWADCM) [reply docs-kits/kits/Demand and Capacity Management Kit/changelog.md:29]: @Frank-Globisch Do not understand, sorry

File: docs-kits/kits/Demand and Capacity Management Kit/changelog.md (hunk 2 above)
Comment (ClosedSourcerer) [reply docs-kits/kits/Demand and Capacity Management Kit/changelog.md]: Addressed by https://github.com/ClosedSourcerer/eclipse-tractusx.github.io/commit/0d23bbf4671cbbe2a57542f4b7188c1d230110e9

File: docs-kits/kits/Demand and Capacity Management Kit/changelog.md (hunk 3 above)
Comment (ClosedSourcerer) [reply docs-kits/kits/Demand and Capacity Management Kit/changelog.md]: Addressed by https://github.com/ClosedSourcerer/eclipse-tractusx.github.io/commit/0d23bbf4671cbbe2a57542f4b7188c1d230110e9

**Review by KZI-CatX — APPROVED** [review]

File: docs-kits/kits/Demand and Capacity Management Kit/changelog.md (hunk 1 above)
Comment (LukasSchmetz) [reply docs-kits/kits/Demand and Capacity Management Kit/changelog.md]: Updated in the md. directly to Demand Volatility

**Review by reginePrivat — APPROVED** [review]

**Review by MWADCM — APPROVED** [review]

File: docs-kits/kits/Demand and Capacity Management Kit/changelog.md (hunk 1 above)
Comment (Frank-Globisch) [reply docs-kits/kits/Demand and Capacity Management Kit/changelog.md]: great, thx.

File: docs-kits/kits/Demand and Capacity Management Kit/changelog.md (hunk 2 above)
Comment (Frank-Globisch) [reply docs-kits/kits/Demand and Capacity Management Kit/changelog.md]: thx for the update.

File: docs-kits/kits/Demand and Capacity Management Kit/changelog.md (hunk 3 above)
Comment (Frank-Globisch) [reply docs-kits/kits/Demand and Capacity Management Kit/changelog.md]: thx for the update.

**Review by Frank-Globisch — APPROVED** [review]
all requested changes covered, LGTM

**Review by sarahmater — APPROVED** [review]

**Pull request merged**